ase/src/main/java/com/ase/ase/rest/controller/BasicCovidController.java

```diff
@@ -38,7 +36,23 @@
                 new HospitalSituationPerDateDto(provinceId, Arrays.asList(
                         new HospitalSituationPerDate("10.10.2020", 12, 10),
                         new HospitalSituationPerDate("12.10.2020", 12, 12))));
+
     }
 
+    @CrossOrigin
+    @GetMapping(value = "/tableData/{province-id}", produces = "application/json")
+    @ResponseBody
+    public ResponseEntity<TableDataPerProvinceDto> listNewTableDataFor(@PathVariable("province-id") int provinceId) {
+        return ResponseEntity.ok(
+                new TableDataPerProvinceDto(provinceId, Arrays.asList(
+                        new TableDataPerDate("10.10.2020", 12, 10, Arrays.asList(
+                                new HospitalSituationPerDate("10.10.2020", 12, 10),
+                                new HospitalSituationPerDate("12.10.2020", 12, 12))),
```

Comment:
Later the Hospital data should be transmitted as sum from the SQL statement, then we do not need a List<HospitalSituationPerDate>